README.md

```diff
@@ -1,162 +1,112 @@
-![GitHub Repo stars](https://img.shields.io/github/stars/opticdev/optic?style=social) ![GitHub contributors](https://img.shields.io/github/contributors-anon/opticdev/optic?style=social) ![npm](https://img.shields.io/npm/dm/@useoptic/openapi-io?style=social) ![license](https://img.shields.io/github/license/opticdev/optic?style=social) 
+![Screenshot 2023-06-14 at 12 55 11 PM](https://github.com/opticdev/optic/assets/5900338/a446bac2-06b6-40bd-a7d4-42f1cc3621ea)
 
-# Optic helps you ship a great API
-
-**Shipping an API is easy** -- the REST is hard. We built Optic because every developer/team should be able to get the benefits of OpenAPI, without all the time/effort/costs. 
-
-[**📋 Documentation for all your APIs**](#document-your-existing-apis-in-minutes)    ← *write your API promises down*
-
-[**🛑 Prevent breaking changes from shipping**](#prevent-breaking-changes-with-api-diffs)  ← *keep your promises*
-
-[**✅ Verify your API is working-as-designed (the OpenAPI and implementation are in sync)**](#verify-your-api-is-working-as-designed)    ← *make sure the API works as-designed*
-
-[**🎨 Build a consistent API that follows your team's standards**](#build-a-consistent-api-that-follows-your-teams-standards)  ← *raise the quality of your API*
+# Optic - the CI tool that improves your APIs
+Optic helps you ship a better API by making it easy for teams to use OpenAPI. Prevent breaking changes, publish accurate documentation and improve the design of your APIs.
 
+**Install**
 ```bash
 npm install -g @useoptic/optic
 ```
+ 
+![GitHub Repo stars](https://img.shields.io/github/stars/opticdev/optic?style=social) ![GitHub contributors](https://img.shields.io/github/contributors-anon/opticdev/optic?style=social) ![npm](https://img.shields.io/npm/dm/@useoptic/openapi-io?style=social) ![license](https://img.shields.io/github/license/opticdev/optic?style=social)
+---
 
-## Document your existing APIs in minutes 
-Use real API traffic to write your initial OpenAPI specification and correctly patch the spec whenever an API changes. 
-
-1. Use the CLI to magically capture traffic `optic capture openapi.yaml https://api.github.com` OR provide a HAR (HTTP Archive format). 
-2. Optic is your API version control tool, like git for APIs. "Undocumented" endpoints are like "Untracked" files in git. Add operations one at a time or use `optic update openapi.yaml --all` to document all of them at once
+## Start locally
 
-**[Documentation: Generate an OpenAPI from traffic](https://www.useoptic.com/docs/openapi/generate-from-traffic)**
+### Prevent breaking changes using `optic diff`
```

Comment:
These sections in general don't tell me much if I want to dig deeper. We should either:
- expand these sections
- link to relevant our docs website (either the CLI reference or the use case / tutorial pages)

Otherwise if I'm interested, I get dead ended.

Reply:
Let's do links. I think that's ultimately what we want anyway (get them on our site)